projects/storefrontlib/layout/main/storefront.component.ts

```diff
@@ -120,8 +120,9 @@ export class StorefrontComponent implements OnInit, OnDestroy {
   }
 
   protected focusOnFirstNavigationItem() {
-    const closestNavigationUi =
-      this.elementRef.nativeElement.querySelector('cx-navigation-ui');
+    const closestNavigationUi = this.elementRef.nativeElement.querySelector(
+      '.NavigationBar cx-navigation-ui'
```

Comment:
Maybe this would work better?

```suggestion
      'header cx-navigation-ui'
```

Reply:
It works very well. I retested on our side. 
If it's okay for you, I would go with this fix.